### PR TITLE
Syntax - comments around ellipses

### LINKED ISSUE #10937: Content bug: invalid syntax in "syntax" sections
## What page(s) did you find the problem on?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/FinalizationRegistry/FinalizationRegistry

## Specific page section or heading?

Syntax

## What is the problem?

[MDN's guidance on writing "Syntax" sections](https://developer.mozilla.org/en-US/docs/MDN/Structures/Syntax_sections) includes two patterns which are invalid syntax:

> ### Callback syntax blocks
>
> [...]
>
> ```js
> filter((currentValue) => { ... } )
> ```
>
> [...]
>
> ### Syntax for arbitrary number of parameters
>
> [...]
>
> ```js
> unshift(element0, element1, ... , elementN)
> ```

Those well-versed in JavaScript likely recognize that a shorthand is in use. In my experience teaching first-time programmers, though, many folks initially assume they are looking at some language feature that they have not yet discovered.

## What did you expect to see?

By extending its recommended patterns with multi-line comments, MDN can avoid suggesting that some functions support distinct invocation grammar. More concretely:

```diff
-filter((currentValue) => { ... } )
+filter((currentValue) => { /* ... */ } )
```

```diff
-unshift(element0, element1, ... , elementN)
+unshift(element0, element1, /* ... , */ elementN)
```

The extra characters might be taken as noise by those who are familiar with the language, but I think the overhead is negigible compared to the clarity it affords newcomers.

If this suggestion is accepted, I'd be glad to update the relevant documentation, templates, and content.

Thanks for maintaining such an invaluable resource!

### PR DESCRIPTION
**Pull request by hamishwillee**
Fixes #10937

@Elchi3 @jugglinmike This updates the syntax guidance for use of ellipses in code blocks and for sequences. It also updates all instances in the web/api tree of ellipses in code blocks.

This does not update the cases for sequences or generally stuff in the javascript tree (e.g. global objects etc). I figure I'll look at those if this goes in.

### CODE REVIEW

File: files/en-us/web/api/worker/postmessage/index.md [hunk 1]
```diff
@@ -134,11 +134,11 @@ self.onmessage = function handleMessageFromMain(msg) {
 buf.byteLength in main BEFORE transfer to worker:        8                     main.js:19
 buf.byteLength in main AFTER transfer to worker:         0                     main.js:27
 
-message from main received in worker:                    MessageEvent { ... }  myWorker.js:3
+message from main received in worker:                    MessageEvent { /* ... */ }  myWorker.js:3
```
Comment (hamishwillee) [inline files/en-us/web/api/worker/postmessage/index.md:137]: Revert

```suggestion
message from main received in worker:                    MessageEvent { ... }  myWorker.js:3
```

File: files/en-us/web/api/worker/postmessage/index.md [hunk 2]
```diff
@@ -134,11 +134,11 @@ self.onmessage = function handleMessageFromMain(msg) {
...
 buf.byteLength in worker BEFORE transfer back to main:   8                     myWorker.js:7
 buf.byteLength in worker AFTER transfer back to main:    0                     myWorker.js:15
 
-message from worker received in main:                    MessageEvent { ... }  main.js:6
+message from worker received in main:                    MessageEvent { /* ... */ }  main.js:6
```
Comment (hamishwillee) [inline files/en-us/web/api/worker/postmessage/index.md:141]: Revert
```suggestion
message from worker received in main:                    MessageEvent { ... }  main.js:6
```

File: files/en-us/web/api/webvr_api/using_the_webvr_api/index.md [hunk 3]
```diff
@@ -142,7 +142,7 @@ Now onto our first WebVR-specific code. First of all, we check to see if {{domxr
       console.log('WebVR 1.1 supported');
 ```
 
-Inside our `if() { ... }` block, we run the {{domxref("Navigator.getVRDisplays()")}} function. This returns a promise, which is fulfilled with an array containing all the VR display devices connected to the computer. If none are connected, the array will be empty.
+Inside our `if() { /* ... */ }` block, we run the {{domxref("Navigator.getVRDisplays()")}} function. This returns a promise, which is fulfilled with an array containing all the VR display devices connected to the computer. If none are connected, the array will be empty.
```
Comment (hamishwillee) [inline files/en-us/web/api/webvr_api/using_the_webvr_api/index.md:145]: Revert 
```suggestion
Inside our `if() { ... }` block, we run the {{domxref("Navigator.getVRDisplays()")}} function. This returns a promise, which is fulfilled with an array containing all the VR display devices connected to the computer. If none are connected, the array will be empty.
```

**Review by Elchi3 — APPROVED** [review]
Great, thanks @hamishwillee 👍

**Pull request merged**